lib/resque/scheduler/engine.rb

```diff
@@ -0,0 +1,48 @@
+require 'resque_web'
+
+module ResqueWeb
+  module Plugins
+    module ResqueScheduler
+      class Engine < Rails::Engine
+        isolate_namespace ResqueWeb::Plugins::ResqueScheduler
+        paths['app'] << 'lib/resque/scheduler/engine/app'
+        paths['app/helpers'] << 'lib/resque/scheduler/engine/app/helpers'
+        paths['app/views'] << 'lib/resque/scheduler/engine/app/views'
+        paths['app/controllers'] << 'lib/resque/scheduler/'\
+          'engine/app/controllers'
+      end
+
+      Engine.routes do
```

Comment:
How different is this from describing routes in a dedicated config/routes.rb file ?